tests/mark/check_cluster_use_metadata.py

```diff
@@ -38,6 +39,19 @@ def function():
         assert len(test_context_list) == 1
         assert test_context_list[0].cluster_use_metadata == cluster_use_metadata
 
+    def check_basic_usage_cluster_spec(self):
+        num_nodes = 200
```

Comment:
just a style nit for the future: we should be consistent and always use a defined test constant or just leave them literals (which is generally ok in tests where they won't end up duplicated or really be significant constants anyway)